velox/functions/prestosql/aggregates/tests/ReduceAggTest.cpp

```diff
@@ -30,6 +31,21 @@ class ReduceAggTest : public functions::aggregate::test::AggregationTestBase {
     allowInputShuffle();
     disableTestStreaming();
   }
+
+  std::vector<RowVectorPtr> fuzzData() {
```

Comment:
nice! How much of this coverage we also get from the aggregation fuzzer?

Reply:
This test found 2 bugs in ReduceAgg::toIntermediate - https://github.com/facebookincubator/velox/pull/6551

Pretty useful I think.